sentry-android-okhttp/build.gradle.kts

```diff
@@ -24,9 +24,7 @@ android {
 
     buildTypes {
         getByName("debug")
-        getByName("release") {
-            consumerProguardFiles("proguard-rules.pro")
-        }
+        getByName("release")
```

Comment:
No, this is fine, because the proguard rules are packaged within `sentry-okhttp` now, and `sentry-okhttp-android` exposes it via `api`, so the rules will be packaged into the final apk anyway